library/core/src/ascii/ascii_char.rs

```diff
@@ -528,8 +528,8 @@ impl AsciiChar {
     }
 
     /// Gets this ASCII character as a byte.
+    #[cfg_attr(flux, flux::spec(fn (Self) -> u8{v: v <= 127}))]
```

Comment:
While you are working on this module: would you mind also adding a spec to `digit_unchecked`? (I already have one ready to go for Kani and, in the spirit of comparing the tools, would be quite keen to see them side-by-side!)

Reply:
will do!